synapse/lib/modelrev.py

```diff
@@ -783,6 +784,292 @@ async def revModel_0_2_26(self, layers):
                 logger.info(f'Updating ndef indexing for {name}')
                 await self._updatePropStortype(layers, prop.full)
 
+    async def revModel_0_2_27(self, layers):
+
+        refforms = []
+        for form in self.core.model.forms.values():
+            for prop in form.props.values():
+
+                if prop.type.name not in ('array', 'it:sec:cpe'):
+                    continue
+
+                if prop.type.isarray and prop.type.arraytype.name != 'it:sec:cpe':
+                    continue
+
```

Comment:
I think you also need to handle props of type `ndef` when they have a value with the `it:sec:cpe` form